test/test_solve_powerflow.jl

```diff
@@ -505,6 +505,11 @@ end
 end
 
 @testset "Test loss factors for larger grid" begin
+    if Sys.iswindows()
+        @test true
+        return
+    end
+
```

Comment:
This is a poor way to fix this, making a test true. This should be implemented as if it isn't windows then run the test 

```suggestion
    if !Sys.iswindows()
@testset "Test loss factors for larger grid" begin

    end
end
```